### DISCUSSION TITLE
🎉 Skeleton has entered Release Candidate status!

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#996] [Announcements]
## :tada: Release Candidate Status :tada:

[Today's update](https://github.com/skeletonlabs/skeleton/discussions/995) marks a huge milestone for the library, after a year in development Skeleton is now officially in release candidate status! We've provided some details below to help inform you of what you might expect and what is still to come.

## What does this mean?

With our [Elements](https://github.com/skeletonlabs/skeleton/discussions/872), [Components](https://github.com/skeletonlabs/skeleton/discussions/928), and now [Utility](https://github.com/skeletonlabs/skeleton/discussions/995) overhauls in place, the library is now feature complete for launch, allowing us to place our full attention on several planned [documentation updates](https://github.com/skeletonlabs/skeleton/issues/783). This is the last major hurdle before the Skeleton v1.0 release. This will introduce a variety of quality improvements to the docs, including a new redesigned homepage, improved page structure, and more. Expect this alongside a small number of [remaining v1 Milestones tasks](https://github.com/skeletonlabs/skeleton/milestone/1).

## Is Skeleton production ready?

Skeleton is not quite production ready, but we're close! While there are no other major breaking changes planned, we do reserve the ability to make last minute adjustments if we deem them necessary. Overall though, the core features of the library should be stable and feature complete. Please be aware we do NOT expect either the release candidate or v1.0 versions of Skeleton to be a bug-free experience. If you find a bug, report it, and we'll resolve it asap! We plan to continue regular maintenance updates per our normal release cadence every 2 weeks.

## What is remaining?

As mentioned above, several [major documentation improvements](https://github.com/skeletonlabs/skeleton/issues/783) are in the works. We're also aiming to release the [autocomplete component](https://github.com/skeletonlabs/skeleton/issues/377), as well as help introduce the [PurgeCSS plugin](https://github.com/skeletonlabs/skeleton/issues/732) created by contributor @AdrianGonz97. Please monitor our [v1.0 Milestones](https://github.com/skeletonlabs/skeleton/milestone/1) if you wish to track all remaining tasks. Expect these to remain in flux over the next couple weeks.

## Launch Day Event

When we feel confident in the overall state of the library, we'll announce the v1.0 launch date. This will include a celebratory launch day event featuring a livestream hosted by myself (Chris) and appearances by several core contributors. During this event we'll have live Q&A and I'll be building a small scale application using Skeleton! Keep an eye on our announcement channels here, on [Discord](https://discord.gg/EXqV7W8MtY), and [Twitter](https://twitter.com/SkeletonUI) if you wish to learn more.

You can follow us on YouTube to be notified when the stream goes live:
https://www.youtube.com/@skeletonlabs

**Comment by SergioJuniorCE** [comment]
LGTM!
